kokoro/continuous.sh

```diff
@@ -5,5 +5,5 @@ set -x
 
 cd github/jib
 
-(cd jib-core; ./gradlew clean build --info)
+(cd jib-core; ./gradlew clean build publishToMavenLocal--info)
```

Comment:
I think you need a space between `publishToMavenLocal` and `--info`

Reply:
Yea haha, fixed.